docs/source/tests/pytest.rst

```diff
@@ -0,0 +1,6 @@
+=====================
+Fixtures with py.test
+=====================
```

Comment:
The `py.test` name is a legacy holdover from, oh, _years_ ago, before `pytest` was a separate project and still lived inside the `py` library.

The `py.test` package on PyPI tells users they should install with `pytest` instead, and was last touched in 2014. I guess technically the current version of `pytest` still supports the old `py.test` alias, but it's become just that: an alias. The tool's name is now `pytest`.

```suggestion
====================
Fixtures with pytest
====================
```